aria_data_loaders/README.md

```diff
@@ -0,0 +1,31 @@
+# Data loaders for Aria data (VRS, MPS)
+
+Project aria already exposes [basic data
+utilities](https://facebookresearch.github.io/projectaria_tools/docs/data_utilities).
+This package implements data-loaders on top of these utilities to make multi-modal data
+access and processing easier.
+
+## Installation
+
+Create the conda env:
+
+```bash
+conda create -n aria_env -y python=3.9
```

Comment:
so this is the another env in addition to the spot-sim2real env right?

Reply:
makes sense - @zephirefaith can we remove this?

Reply:
The intention here was to keep "stand-alone" instructions as well as Spot-Sim2Real instructions. The basic Aria data-tools implemented here are pretty independent of spot library for usage. I will add another heading here which explicitly states that the Stand-alone use-case of Aria tools needs new env creation, just install aria-tools into spot-sim2real otherwise. How does that sound? @jimmytyyang @rutadesai @KavitShah1998

Reply:
What do we think is the role of aria_data_utils in the long-term? Should we factor it out or reuse it for "episodic VRS memories"